WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionPreviewViewModel.cs

```diff
@@ -163,7 +163,16 @@ private async Task OnExportPsbtAsync()
 	{
 		if (Transaction is { })
 		{
-			var saved = await TransactionHelpers.ExportTransactionToBinaryAsync(Transaction);
+			bool saved = false;
+			try
+			{
+				saved = await TransactionHelpers.ExportTransactionToBinaryAsync(Transaction);
+			}
+			catch (Exception ex)
+			{
+				Logger.LogError(ex);
+				await ShowErrorAsync("Transaction Building", ex.ToUserFriendlyString(), "Wasabi was unable to export the PSBT.");
```

Comment:
I think we should not say `Transaction Building` here.

```suggestion
				await ShowErrorAsync("Transaction Export", ex.ToUserFriendlyString(), "Wasabi was unable to export the PSBT.");
```